extensions/BMO/lib/Reports/WhatsNext.pm

```diff
@@ -111,32 +111,34 @@ sub get_tracking_status_flags {
   my $beta_name    = "cf_tracking_firefox$beta_major";
   my $release_name = "cf_tracking_firefox$release_major";
 
-  my $nightly_field = Bugzilla::Extension::TrackingFlags::Flag->new({name => $nightly_name});
-  my $beta_field    = Bugzilla::Extension::TrackingFlags::Flag->new({name => $beta_name});
-  my $release_field = Bugzilla::Extension::TrackingFlags::Flag->new({name => $release_name});
+  my $nightly_field
+    = Bugzilla::Extension::TrackingFlags::Flag->new({name => $nightly_name});
+  my $beta_field
+    = Bugzilla::Extension::TrackingFlags::Flag->new({name => $beta_name});
+  my $release_field
+    = Bugzilla::Extension::TrackingFlags::Flag->new({name => $release_name});
   return {} if !$nightly_field && !$beta_field && !$release_field;
 
-  $flag_data->{tracking} = {
-    nightly => $nightly_field,
-    beta    => $beta_field,
-    release => $release_field,
-  };
+  $flag_data->{tracking}
+    = {nightly => $nightly_field, beta => $beta_field, release => $release_field,
+    };
```

Comment:
Please revert these formatting changes that drop the `=` onto a new line; that isn't in alignment with our perlstyle config, nor is it used anywhere else.